test/cadet/accounts/accounts_test.exs

```diff
@@ -0,0 +1,147 @@
+defmodule Cadet.AccountsTest do
+  @moduledoc """
+  Some tests in this module use pre-recorded HTTP responses saved by ExVCR.
+  this allows testing without the use of actual external IVLE API calls.
+
+  In the case that you need to change the recorded responses, you will need
+  to set the two environment variables IVLE_KEY (used as a module attribute in
+  `Cadet.Accounts.IVLE`) and TOKEN (used here). Don't forget to delete the
+  cassette files, otherwise ExVCR will not override the cassettes.
+  """
+
+  use Cadet.DataCase
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  alias Cadet.Accounts
+  alias Cadet.Repo
+  alias Cadet.Accounts.Query
+
+  @token String.replace(inspect(System.get_env("TOKEN")), ~s("), "")
+
+  setup_all do
+    HTTPoison.start()
+  end
+
+  test "create user" do
+    {:ok, user} =
+      Accounts.create_user(%{
+        name: "happy user",
+        role: :student
+      })
+
+    assert user.name == "happy user"
```

Comment:
Use pattern matching instead, e.g. `assert %{name: "happy user", role: :student} = user`. Similarly for the rest

Reply:
Done with 735169f.